tests/iostability/test_iostability_happy_path.py

```diff
@@ -57,19 +58,40 @@ def setup_class(cls):
                 cls.master_node_list.append(node_obj)
             else:
                 cls.worker_node_list.append(node_obj)
+        cls.health_obj = Health(cls.master_node_list[0]["hostname"],
+                                cls.master_node_list[0]["username"],
+                                cls.master_node_list[0]["password"])
         cls.test_cfg = configmanager.get_config_wrapper(fpath="config/iostability_test.yaml")
         cls.setup_type = CMN_CFG["setup_type"]
         cls.test_completed = False
         cls.iolib = IOStabilityLib()
+        cls.duration_in_days = os.getenv("DURATION_OF_TEST_IN_DAYS",
+                                         cls.test_cfg['happy_path_duration_days'])
+
+        cls.clients = os.getenv("CLIENT_SESSIONS_PER_WORKER_NODE", 0)
+        if cls.clients == 0:
+            if cls.setup_type == 'HW':
```

Comment:
Please refer to earlier similar comment. Add if appropriate.

Reply:
Default value will be VM. please let me know if we need to switch to HW client values as default case